### DISCUSSION TITLE
Can't get reverified as an Educator

### DISCUSSION THREAD

**Discussion by MrZebarth** [community/community#52664] [GitHub Education]
I've been using GitHub for years in my classroom and I'm a GitHub Campus Advisor. However, I've been given two weeks to reverify and have been rejected 4 times in a row now for reasons that make no sense.
![Screenshot_20230412-170755](https://user-images.githubusercontent.com/5191364/231586231-59258c1a-baee-445e-92aa-d7081b3f442e.png)

I've completed my bio, my staff card clearly mentions my school (I would attach a photo, but don't want to break privacy guidelines), and I picked the campus off a provided map and have been there for 17 years. It hasn't moved!

And now all help for verification issues have been relegated to the online community, which I'm really not sure can help much in this regard. 

Can anyone help me get through this verification process??

**Comment by opegamanalitarok** [comment]
I'm sorry to hear that you're experiencing issues with the GitHub Campus Advisor verification process. Here are some steps you can take to try and resolve this issue:

1. Double-check your verification information: Make sure that all the information you provided during the verification process is accurate and up-to-date. Ensure that your bio, staff card, and campus location are all correct and clearly stated.

2. Reach out to GitHub support: If you have been rejected multiple times and you're not sure why, you can reach out to GitHub support for help. They may be able to provide more specific feedback on what you need to do to pass the verification process.

3. Connect with other Campus Advisors: Reach out to other GitHub Campus Advisors who have successfully gone through the verification process. They may be able to provide insight into what worked for them and offer tips and advice to help you succeed.

4. Attend a Campus Advisor training session: If there are any upcoming Campus Advisor training sessions in your area, consider attending one. These sessions can provide valuable information on how to successfully navigate the verification process and become a more effective Campus Advisor.

5. Be patient: Verification processes can take time and require patience. Keep working on your verification information and reach out to GitHub support if you need additional help.

I hope these suggestions help you successfully navigate the Campus Advisor verification process. Good luck!

**Reply by MrZebarth** [reply]
I've tried multiple things and got rejected without cause the last time. Absolutely no reasons were given for not approving. I filed a help desk ticket and it was immediately closed and they told me to direct all questions to the forum. This is beyond frustrating and making me reconsider using GitHub with my classes

**Comment by samus-aran** [comment]
Hi @MrZebarth 👋 

Do you have a screenshot of the rejection reasons? You should be able to see your latest submission via [this page](https://education.github.com/discount_requests/pack_application).
